views/download.html.twig

```diff
@@ -1,64 +1,120 @@
 {% extends "layout.html.twig" %}
 
 {% block content %}
-    <h1>Download Composer <span class="latest">Latest: v{{ latestStable }}</span></h1>
 
-    {% if windows %}
-        <h2>Windows Installer</h2>
-        <p>The installer will download composer for you and set up your PATH environment variable so you can simply call <code>composer</code> from any directory.</p>
-        <p>Download and run <a href="/Composer-Setup.exe">Composer-Setup.exe</a> - it will install the latest composer version whenever it is executed.</p>
+    <article role="article">
 
-        <h2>Command-line installation</h2>
-    {% endif %}
+        <h1 role="heading">Download Composer <span class="latest">Latest: v{{ latestStable }}</span></h1>
 
-    <p>To quickly install Composer in the current directory, run the following script in your terminal. To automate the installation, use <a href="/doc/faqs/how-to-install-composer-programmatically.md">the guide on installing Composer programmatically</a>.</p>
-    <pre class="installer">php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+        {% if windows %}
+            <h2 role="heading">Windows Installer</h2>
+            <p>The installer will download composer for you and set up your PATH environment variable so you can simply
+                call <code>composer</code> from any directory.</p>
+            <p>
+                Download and run <a href="/Composer-Setup.exe"
```

Comment:
This weird wrapping makes everything a lot *less* readable IMO

Reply:
See above comment regarding line wrapping so we get out of this mouse and cat game. :)